ipi/engine/motion/dynamics.py

```diff
@@ -276,6 +311,8 @@ def bind(self, motion):
         self.fixcom = motion.fixcom
         self.fixatoms = motion.fixatoms
         self.enstype = motion.enstype
+        if motion.enstype in EDA.integrators:
+            self.eda = motion.eda
```

Comment:
there are way too many of these if statements. everything suggests to me this should be a separate motion class, like planetary or constrained dynamics

Reply:
These `if` statements no longer exists since the "driven dynamics/EDA" code has been moved into the `DrivenDynamics` class. The only statements of this kind are still in `engine/properties.py`.

Reply:
I still see it here though....

Reply:
ping. what's going on? please bring this back to a clean dynamics.py file.